easybuild/easyconfigs/s/Spyder/Spyder-6.0.1-gfbf-2023b.eb

```
@@ -0,0 +1,118 @@
+
+exts_list = [
+    ('asyncssh', '2.17.0', {
+        'checksums': ['3b159c105aa388c1e2245c4faf483f540ada8cad99402281119100166e5edb3c'],
+    }),
+    ('diff-match-patch', '20230430', {
+        'checksums': ['953019cdb9c9d2c9e47b5b12bcff3cf4746fc4598eb406076fa1fc27e6a1f15c'],
+    }),
+    ('importlib_metadata', '8.4.0', {
+        'checksums': ['9a547d3bc3608b025f93d403fdd1aae741c24fbb8314df4b155675742ce303c5'],
+    }),
+    ('pygithub', '2.4.0', {
+        'modulename': 'github',
+        'checksums': ['6601e22627e87bac192f1e2e39c6e6f69a43152cfb8f307cee575879320b3051'],
+    }),
+    ('pylint_venv', '3.0.3', {
+        'checksums': ['df12a17fca39a94acc1c9a0f1dcf68141e90fe685569d78c046695c67c4e55fa'],
+    }),
+    ('pyls-spyder', '0.4.0', {
+        'checksums': ['45a321c83f64267d82907492c55199fccabda45bc872dd23bf1efd08edac1b0b'],
+    }),
+    ('pyuca', '1.2', {
+        'checksums': ['8a382fe74627f08c0d18908c0713ca4a20aad5385f077579e56208beea2893b2'],
+    }),
+    ('pyxdg', '0.28', {
+        'modulename': 'xdg',
+        'checksums': ['3267bb3074e934df202af2ee0868575484108581e6f3cb006af1da35395e88b4'],
+    }),
+    ('QDarkStyle', '3.2.3', {
+        'checksums': ['0c0b7f74a6e92121008992b369bab60468157db1c02cd30d64a5e9a3b402f1ae'],
+    }),
+    ('inflection', '0.5.1', {
+        'checksums': ['1a29730d366e996aaacffb2f1f1cb9593dc38e2ddd30c91250c6dde09ea9b417'],
+    }),
+    ('qstylizer', '0.2.3', {
+        'checksums': ['5f2f5eb2c65c6c45e950462b565fcaa5107a91d1cfc1a85b8e0d831d727ee8f6'],
+    }),
+    ('QtAwesome', '1.3.1', {
+        'checksums': ['075b2c9ee01cbaf5e3a4bebed0e5529ee8605981355f21dea051b15c1b869e1b'],
+    }),
+    ('spyder_kernels', '3.0.0', {
+        'checksums': ['3d316ea04802ed1feb366405cca03df2349dc9e35ad8653538d2fe48c0e9417f'],
+    }),
+    ('superqt', '0.6.6', {
+        'checksums': ['792e09165c8a788ee245bdb784e018f9077fb309253354d86793cdf1d092f99f'],
+    }),
+    ('textdistance', '4.6.3', {
+        'checksums': ['d6dabc50b4ea832cdcf0e1e6021bd0c7fcd9ade155888d79bb6a3c31fce2dc6f'],
+    }),
+    ('three-merge', '0.1.1', {
+        'checksums': ['60f6afe144595560d63ae32625351bcef3b94733b54eb97800a9feb0f3d9d970'],
+    }),
+    ('watchdog', '5.0.3', {
+        'checksums': ['108f42a7f0345042a854d4d0ad0834b741d421330d5f575b81cb27b883500176'],
+    }),
+    ('jellyfish', '1.1.0', {
+        'checksums': ['2a2eec494c81dc1eb23dfef543110dad1873538eccaffabea8520bdac8aecbc1'],
+    }),
+    ('platformdirs', '3.11.0', {
+        'checksums': ['cf8ee52a3afdb965072dcc652433e0c7e3e40cf5ea1477cd4b3b1d2eb75495b3'],
+    }),
+    (name, version, {
+        'source_tmpl': '%(namelower)s-%(version)s.tar.gz',
+        'preinstallopts': local_preinstallopts,
+        'checksums': ['709782e88091588bbe614de6ebbde7b47544a4d6c225e199de5ad22b77d8b130'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/spyder'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
```

With https://github.com/easybuilders/easybuild-easyblocks/pull/3477 this can be simply:
```suggestion
enhance_sanity_check = True
sanity_check_paths = {'files': ['bin/spyder']}
```

I'd still use that already even though the directory won't be checked for but at least one of the import checks would certainly fail without this folder.